src/uwtools/resources/rocoto.jsonschema

```diff
@@ -0,0 +1,70 @@
+{
+    "$defs": {
+        "dateTimePattern": {
+            "type": "string",
+            "pattern": "^(\\d{12}) (\\d{12}) (\\d{2}):(\\d{2}):(\\d{2})$"
+        }
+    },
+    "type": "object",
+    "properties": {
+        "workflow": {
+            "type": "object",
+            "properties": {
+                "attrs": {
+                    "type": "object",
+                    "properties": {
+                        "realtime": {
+                            "type": "boolean"
+                        },
+                        "scheduler": {
+                            "type": "string",
+                            "enum": [
+                                "sge",
+                                "lsf",
+                                "lsfcray",
+                                "ll",
+                                "torque",
+                                "moabtorque",
+                                "moab",
+                                "pbspro",
+                                "slurm",
+                                "cobalt"
+                            ]
+                        },
+                        "cyclethrottle": {
+                            "type": "integer",
+                            "minimum": 0
+                        },
+                        "taskthrottle": {
+                            "type": "integer",
+                            "minimum": 0
+                        }
+                    },
+                    "required": ["realtime", "scheduler"]
+                },
+                "cycledefs": {
+                    "type": "object",
+                    "properties": {
+                        "groupname": {
```

Comment:
I am curious about this entry...the `cycledefs` section can include any number of arbitrarily defined entries.

Reply:
I updated the valid `hello_workflow.yaml` to account for various different sections we could see within the `cycledef` and the tests pass accordingly. Let me know if you feel like this is sufficient. @christinaholtNOAA